tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/PublicAssetSetMethodContract.scala

```diff
@@ -1963,4 +1972,106 @@ trait PublicAssetSetMethodContract {
       .jsonPath()
       .get("methodResponses[0][1].created.4f29.id")
   }
+
+  @Test
+  def createShouldReturnFailWhenPublicAssetQuotaLimitIsExceeded(): Unit = {
+    val content = "Your asset content here".repeat(10).getBytes
+    val uploadResponse: UploadResponse = uploadAsset(content)
+    val uploadResponse2: UploadResponse = uploadAsset(content)
+    val uploadResponse3: UploadResponse = uploadAsset(content)
+
+    val request: String =
```

Comment:
Spent time to try understand logic of this test
IMO we can 2 requests, 
1 created when not quota exceed 
1 notCreated when quota exceed